src/Item_Devices.php

```diff
@@ -684,16 +686,18 @@ public static function showForItem(CommonGLPI $item, $withtemplate = 0)
                 $table_options['itemtype'] = $itemtype;
                 $link                      = getItemForItemtype(static::getType());
```

Comment:
```suggestion
                $link                      = getItemForItemtype(static::class);
```

`getType` has no real purpose and it probably confuses PHPStan. You can use `::class` on everything except object instances on the 10.0 branch. `$item::class` will work on `main` but for 10.0 you could use `get_class($item)`.